docs/release-notes/1.4.0-preview/release-1.4.0-preview.md

```diff
@@ -0,0 +1,88 @@
+# [ML.NET](http://dot.net/ml) 1.4.0-preview
+
+## **New Features**
+- **Deep Neural Networks Training (0.16.0-preview)** ([#4151](https://github.com/dotnet/machinelearning/pull/4151))  
+
+  Improves the in-preview `ImageClassification` API further:
+  - Increases DNN training speed by ~10x.
+  - Prevents repeated computations by caching featurized image values to disk from intermediate layers to train the final fully-connected layer.
+  - Reduced and constant memory footprint.
+  - Simplifies the API by not requiring the user to pre-process the image.
+  - Introduces callback to provide metrics during training such as accuracy, cross-entropy.
+  - Improved image classification sample.
+
+  ```cs
+        public static ImageClassificationEstimator ImageClassification(
+            this ModelOperationsCatalog catalog,
+            string featuresColumnName,
+            string labelColumnName,
+            string scoreColumnName = "Score",
+            string predictedLabelColumnName = "PredictedLabel",
+            Architecture arch = Architecture.InceptionV3,
+            int epoch = 100,
+            int batchSize = 10,
+            float learningRate = 0.01f,
+            ImageClassificationMetricsCallback metricsCallback = null,
+            int statisticFrequency = 1,
+            DnnFramework framework = DnnFramework.Tensorflow,
+            string modelSavePath = null,
+            string finalModelPrefix = "custom_retrained_model_based_on_",
+            IDataView validationSet = null,
+            bool testOnTrainSet = true,
+            bool reuseTrainSetBottleneckCachedValues = false,
+            bool reuseValidationSetBottleneckCachedValues = false,
+            string trainSetBottleneckCachedValuesFilePath = "trainSetBottleneckFile.csv",
+            string validationSetBottleneckCachedValuesFilePath = "validationSetBottleneckFile.csv"
+            )
+
+  ```
+
+  [Design specification](https://github.com/dotnet/machinelearning/blob/cd591dd492833964b6829e8bb2411fb81665ac6d/docs/specs/DNN/dnn_api_spec.md)
+
+  [Sample](https://github.com/dotnet/machinelearning/blob/master/docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs)
+
+- **Database Loader (0.16.0-preview)** ([#4070](https://github.com/dotnet/machinelearning/pull/4070),[#4091](https://github.com/dotnet/machinelearning/pull/4091),[#4138](https://github.com/dotnet/machinelearning/pull/4138))  
+
+  Additional DatabaseLoader support:
+  -  Support DBNull.
+  -  Add `CreateDatabaseLoader<TInput>` to map columns from a .NET Type.
+  -  Read multiple columns into a single vector
+
+  [Design specification](https://github.com/dotnet/machinelearning/pull/3857) 
+  
+  [Sample](https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/DatabaseLoaderTests.cs)  
+
+  ```cs
+  public static DatabaseLoader CreateDatabaseLoader(this DataOperationsCatalog catalog,
```

Comment:
As example code, this would be better understood by users:

```
string connectionString = "YOUR_RELATIONAL_DATABASE_CONNECTION_STRING";

string commandText = "SELECT * from URLClicks";

DatabaseLoader loader = mlContext.Data.CreateDatabaseLoader<UrlClick>();
            
DatabaseSource dbSource = new DatabaseSource(SqlClientFactory.Instance, 
                                                connectionString, 
                                                commandText);         
IDataView dataView = loader.Load(dbSource);
```